tests/log_color/Makefile

```diff
@@ -0,0 +1,10 @@
+include ../Makefile.tests_common
+
+USEMODULE += log_color
+
+TEST_ON_CI_WHITELIST += all
```

Comment:
@cladmi this is default now, right?

Reply:
Should be since #11680. @aabadie please remove this line, you can directly squash.

Reply:
Apparently yes
```
makefiles/murdock.inc.mk:TEST_ON_CI_WHITELIST ?= all
```
I'll remove this line then.

Reply:
Done